dbus-tokio/Cargo.toml

```diff
@@ -14,14 +14,14 @@ readme = "README.md"
 
 [dependencies]
 dbus = { path = "../dbus", version = "0.9", features=["futures"] }
-mio = "0.6.21"
 libc = "0.2.69"
-tokio = {version = "0.2.4", features=["time", "macros", "io-driver", "rt-core", "rt-util", "rt-threaded", "stream"]}
+tokio = {version = "1.0", features=["time", "net"]}
+parking_lot = "0.11"
```

Comment:
Unless there is a strong reason not to use std's Mutex, I would prefer to avoid additional dependencies.